assets/data/social.yml

```diff
@@ -495,15 +497,39 @@ bilibili:
 
 # 063: Email
 email:
-  Weight: 63
+  Weight: -4
   Template: mailto:%v
   Title: Email
   Icon:
-    Class: far fa-envelope fa-fw
+    Class: fa fa-envelope fa-fw
+
+# 064: Phone
+phone:
+  Weight: -3
+  Template: tel:%v
+  Title: Phone
+  Icon:
+    Class: fa fa-phone fa-fw
+
+# 065: PGP
+pgp:
```

Comment:
I think there should be an option to have the key hosted on own site (i.e. the Hugo site), too; one can just put the key in `static/` instead of uploading it onto a third-party server.

Reply:
Obviously this will be the best.
Maybe, we configure it like Mastodon.
Example:

```
[params.social.pgp]
    id = "0x5AEB729B3E642A16"
   
    # Point to a keyserver
    prefix = "https://pgp.key-server.io/"
  
   # Relative to my site
   # Assume the key hosted at https://example.com/0x5AEB729B3E642A16
   # prefix = "httsp://example.com"
```

Reply:
Makes sense, yes.